content/php/concepts/string-functions/terms/explode/explode.md

```diff
@@ -0,0 +1,61 @@
+---
+Title: 'explode()'
+Description: 'Splits a given string by a delimiter and returns an array of the substrings produced.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `explode()` function takes a string and splits it on a given delimiter string, and returns an array of the substrings produced.
+
+## Syntax
+
+```pseudo
+explode($delimiter, $string, $limit)
+```
+
+The `$delimiter` parameter specifies the string on which `$string` will be split.
+The `$string` parameter is the string that will be split by the `explode()` function.
+
+The `$limit` parameter is an optional `int` that specifies how many strings will be returned in the resulting array as follows:
+
+- A positive value will return up to `$limit` strings, if more than `$limit` strings can be produced splitting on `$delimiter`, the last element will contain the remainder of `$string`.
+- A `$limit` of zero is treated as 1.
+- A negative `$limit` will omit the last -`$limit` elements from the result.
```

Comment:
```suggestion
- A negative `$limit` will omit the last n elements from the result.
```

Reply:
I found this description confusing, perhaps revise to something similar to the suggestion above